src/EnergyPlus/CurveManager.cc

```diff
@@ -239,10 +239,10 @@ namespace Curve {
                    this->coeff[5] * V1 * V2 + this->coeff[6] * V1 * V1 * V1 + this->coeff[7] * V2 * V2 * V2 + this->coeff[8] * V1 * V1 * V2 +
                    this->coeff[9] * V1 * V2 * V2 + this->coeff[10] * V1 * V1 * V2 * V2 + this->coeff[11] * V3 * V2 * V2 * V2;
         case CurveType::TriQuadratic: {
-            auto const &c(this->coeff);
-            Real64 const V1s(V1 * V1);
-            Real64 const V2s(V2 * V2);
-            Real64 const V3s(V3 * V3);
+            auto const &c = this->coeff;
```

Comment:
Getting rid of constructor notation which messes up `grep` scripts and is stupid.